LayoutTests/fast/text-extraction/debug-text-extraction-ignore-autofilled-fields.html

```diff
@@ -0,0 +1,52 @@
+<!-- webkit-test-runner [ useFlexibleViewport=true textExtractionEnabled=true ] -->
+<!DOCTYPE html>
+<html>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta charset="utf-8">
+<head>
+<style>
+body {
+    white-space: pre-wrap;
+}
+
+input {
+    font-size: 18px;
+}
+</style>
+<script src="../../resources/ui-helper.js"></script>
+</head>
+<body>
+<div><input type="email" placeholder="Enter your email" /></div>
+<div><input type="password" placeholder="Enter your password" /></div>
```

Comment:
Nit, since it stands out compared to the number input below 😅 
```suggestion
<div><input type="email" placeholder="Enter your email"></div>
<div><input type="password" placeholder="Enter your password"></div>
```

Reply:
Whoops! Fixed.